plugins/modules/dss_general_settings.py

```diff
@@ -119,21 +130,34 @@
     add_dataikuapi_to_path,
     update,
     exclude_keys,
+    smart_update_named_lists
 )
 
+smart_update_fields_template = {
+    "containerSettings": {
+        "executionConfigs": []
+    },
+    "sparkSettings": {
+        "executionConfigs": []
+    }
+}
 
 encrypted_fields = [
     "ldapSettings.bindPassword", "ssoSettings.samlSPParams.keystorePassword", "ssoSettings.openIDParams.clientSecret",
     "azureADSettings.credentialsClientSecret", "azureADSettings.credentialsCertificatePassword"
 ]
+smart_update_fields = [
+    "containerSettings.executionConfigs", "sparkSettings.executionConfigs"
+]
```

Comment:
Could be built from `smart_update_fields_template`

Reply:
refactored in [0bfb489](https://github.com/dataiku/dataiku-ansible-collection/pull/5/commits/0bfb489462c9ff843fd81576dc7c039c21f5eead)[0bfb489](https://github.com/dataiku/dataiku-ansible-collection/pull/5/commits/0bfb489462c9ff843fd81576dc7c039c21f5eead)